webapp/components/admin_console/ldap_settings.jsx

```diff
@@ -391,7 +391,7 @@ export default class LdapSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.ldap.loginNameTitle'
-                            defaultMessage='Login Field Name:'
+                            defaultMessage='Login Field Default Text:'
```

Comment:
Should we use "sign-in" to match the UI, rather than "login" which isn't used in the UI? 

![image](https://cloud.githubusercontent.com/assets/177788/16421431/c56cb71a-3d09-11e6-9ae9-e97574e01578.png)